client/dist/styles.css

```diff
@@ -49,3 +49,38 @@
    max-width: 1280px;
    margin: auto;
  }
+
+ /* Star styling  */
+ .ratings {
+  position: relative;
+  vertical-align: middle;
+  display: inline-block;
+  color: #b1b1b1;
+  overflow: hidden;
+}
+.full-stars {
+  position: absolute;
+  left: 0;
+  top: 0;
+  white-space: nowrap;
+  overflow: hidden;
+  color: #fde16d;
+}
+.empty-stars:before, .full-stars:before {
+  content:"\2605\2605\2605\2605\2605";
```

Comment:
Are the stars an imported image or was it something created in css?

Reply:
It's unicode character for [a star](https://unicode-table.com/en/2605/)